### PR TITLE
Remove AddDefaultValueSerializer when loading resources from the DB

### PR DESCRIPTION
**Pull request by MarisaGoergen**
The DefaultValueAttributeProvider.Handle function replaces system default values with the costum default values set using the DefaultValueAttribute. The consequence was that setting the system default value wasn't possible anymore, since it gets overwritten when loading the resource  from the database. NewtonsoftJson is able to set the costum default value directly when populating an object since Version 12.0.1.

### CODE REVIEW

**Comment by MarisaGoergen** [comment]
Is the `DefaultValueAttributeProvider` still neccessary? @Toxantron

**Comment by Toxantron** [comment]
> Is the `DefaultValueAttributeProvider` still neccessary? @Toxantron

Yes, we still need it (and hopefully use it) to populate config prototypes when sending them to the UI.

File: src/Moryx.Resources.Management/Resources/ResourceEntityAccessor.cs [hunk 1]
```diff
@@ -78,8 +78,7 @@ public Resource Instantiate(IResourceTypeController typeController, IResourceGra
 
             // Read everything else from defaults
             ValueProviderExecutor.Execute(Instance, new ValueProviderExecutorSettings()
-                .AddFilter(new DataMemberAttributeValueProviderFilter(false))
-                .AddDefaultValueProvider());
+                .AddFilter(new DataMemberAttributeValueProviderFilter(false)));
```
Comment (Toxantron) [inline src/Moryx.Resources.Management/Resources/ResourceEntityAccessor.cs:81]: I am wondering whether we need this call at all now. Does it still do anything without a registered provider? Does it populate `null` objects for example?

**Comment by MarisaGoergen** [comment]
That's what `DefaultValueHandling.IgnoreAndPopulate` is for.

**Comment by Toxantron** [comment]
> That's what `DefaultValueHandling.IgnoreAndPopulate` is for.

That's what I had in mind, just wanted to check.

File: src/Moryx.Resources.Management/Resources/ResourceEntityAccessor.cs (hunk 1 above)
Comment (MarisaGoergen) [reply src/Moryx.Resources.Management/Resources/ResourceEntityAccessor.cs:81]: Really is not needed anymore, since it should do nothing

**Review by Toxantron — APPROVED** [review]

**Pull request merged**